ui/packages/consul-ui/app/templates/dc/services/instance.hbs

```diff
@@ -129,6 +135,8 @@ as |item|}}
             </h1>
             <Consul::ExternalSource @item={{item}} @withInfo={{true}} />
             <Consul::Kind @item={{item}} @withInfo={{true}} />
+            {{! TODO: Looks like we can get this straight from item.Proxy.Mode }}
+            {{! the less we need `proxy` the better }}
             {{#if (eq proxy.ServiceProxy.Mode 'transparent')}}
               <Consul::TransparentProxy />
             {{/if}}
```

Comment:
I noticed here that we also have access to `Mode` on the `item` (the service instance) as well as the `proxy`. The less we use proxy the better. Actually, glad I self reviewed this I need to look at this a little more 🤔  brb

Reply:
After I looked more at this I realized that I needed to also swap our `proxy` for `meta` here and in a few other places. I've added another 4 commits on the end here to cover this (from `Make TProxy...` cc698d304fe2b9389f8264066084adaf3469bae1 to `Document CONSUL_...` 38ba3c0224a4c002256da128dec9a46c40ec5ba6)